src/module.ts

```diff
@@ -100,6 +101,8 @@ export default defineNuxtModule<ModuleOptions>({
           compatibility_flags: ['nodejs_compat']
         })
       }
```

Comment:
I would also move this code within the `setupCloudflare()` function.

Are you happy to move `src/cloudflare.ts` to `src/hosting/cloudflare.ts`